dash/testing/plugin.py

```diff
@@ -41,19 +41,6 @@ def pytest_addoption(parser):
         "--headless", action="store_true", help="set this flag to run in headless mode"
     )
 
-    dash.addoption(
-        "--percy-assets",
-        action="store",
-        default="tests/assets",
-        help="configure how Percy will discover your app's assets",
-    )
```

Comment:
Does the new percy runner have a different way of collecting assets so it doesn't need this?

Reply:
I think that was specific to the old percy client, can't find something similar for the new one. Do you think it would be better to make the migrate in a separate PR with a proper percy baseline build to compare ?

Reply:
If they can make do without that it's way better... let's just verify that assets like images *are* in fact making it into the snapshots, and keep going.